fmriprep/workflows/base.py

```diff
@@ -133,8 +146,16 @@ def init_single_subject_wf(subject_id, task_id, name,
 
     workflow.connect([
         (inputnode, anat_preproc_wf, [('subjects_dir', 'inputnode.subjects_dir')]),
+        (bidssrc, bids_info, [(('t1w', fix_multi_T1w_source_name), 'in_file')]),
+        (bidssrc, summary, [('t1w', 't1w'),
+                            ('t2w', 't2w'),
+                            ('bold', 'bold')]),
```

Comment:
This currently provides the name of all BOLD series, rather than just the ones in the current run.

This makes this node more stable when running different task IDs in different processes, but it does mean that if someone doesn't choose to run FMRIPREP on all of their data, then the summary may not match the rest of the outputs.

To make it show just the images in the current execution, we'd just set `summary.inputs.bold = subject_data['bold']`